pkg/query-service/model/v3/v3.go

```diff
@@ -415,20 +415,20 @@ func (f *FilterSet) Validate() error {
 }
 
 type FilterItem struct {
-	Key      string      `json:"key"`
-	Value    interface{} `json:"value"`
-	Operator string      `json:"op"`
+	Key      AttributeKey `json:"key"`
+	Value    interface{}  `json:"value"`
+	Operator string       `json:"op"`
 }
 
 type OrderBy struct {
-	ColumnName string `json:"columnName"`
-	Order      string `json:"order"`
+	ColumnName AttributeKey `json:"columnName"`
```

Comment:
Why does the order by `ColumnName` need to be `AttributeKey`?

Reply:
We are allowing tags to be extracted as a column.

Reply:
And how is that related to order by clause?

Reply:
User can orderBy any tag or column or resource attribute so we need to use `AttributeKey `

Reply:
The Order by clause will either be aggregation result + (any number of the `groupBy` attributes). My question is directed around that. Why/How will the aggregation result fit into the `AttributeKey` definition? Since this is just ordering of SELECT clause, result why does the frontend need to give the input in
```
type AttributeKey struct {
	Key      string               `json:"key"`
	DataType AttributeKeyDataType `json:"dataType"`
	Type     AttributeKeyType     `json:"type"`
	IsColumn bool                 `json:"isColumn"`
}
```
?